colossus-metrics/src/main/scala/colossus/metrics/MetricSystem.scala

```diff
@@ -82,7 +82,7 @@ object MetricSystem {
    * @return
    */
   def apply(namespace: MetricAddress, collectionIntervals: Seq[FiniteDuration] = Seq(1.second, 1.minute),
-            collectSystemMetrics: Boolean = true, config : Config = ConfigFactory.defaultReference())
+            collectSystemMetrics: Boolean = true, config : Config = ConfigFactory.load())
```

Comment:
After reading typesafe config's docs, they recommended having libraries call this load function.  From what I gather from Akka's source, they call this former function. I switched as, I would rather take the advice from the library creator, rather than a user of  the library